release/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.11.5
```

Comment:
I guess this is fine for the moment, but we might consider using other base images (like the stuff from distroless e.g.)